components/templates/TerminalChallenge/Info.tsx

```diff
@@ -0,0 +1,21 @@
+import { CopyableCode } from 'components/ui/CopyableCode'
+
+export const Info = ({
+  genesis,
+  copyableCodeContent
+}: {
+  genesis: any,
+  copyableCodeContent: string
+}) => {
+  return (
+    <div className="flex">
+      <div className="px-1 py-6 sm:px-12 text-white">
+        <div
+          className="genesis"
+          dangerouslySetInnerHTML={{ __html: genesis.body.html }}
+        ></div>
+        <CopyableCode code={copyableCodeContent} language='shell' />
+      </div>
```

Comment:
In #56, we introduce tips for this lesson. How can we integrate this in a dynamic way? It can be a follow-up PR.

Reply:
I have added 3 props for handling the `Tips` section.
`tips`: The main message to be displayed within the tip
`tipTitle`: The title of the tip
`TipAdditionalContent`: This is an React FC which can be passed down from the main challenge component down to the template component

I have kept all the above mentioned props as optional, in case a page doesn't require to display any tips